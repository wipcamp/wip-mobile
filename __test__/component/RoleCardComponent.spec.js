/**
 * @jest-environment node
 */

import React from 'react'
import renderer from 'react-test-renderer'

import RoleCard from '../../components/RoleCardComponent'

it('rendering', () => {
    const tree = renderer
        .create(
            <RoleCard
                key={1}
                ios="test"
                android="test"
                role="test"
            />
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})