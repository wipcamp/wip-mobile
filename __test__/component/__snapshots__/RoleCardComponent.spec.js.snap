// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      Object {
        "flex": 0.2,
      }
    }
  />
  <View
    style={
      Object {
        "flex": 0.2,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={false}
      android="test"
      ellipsizeMode="tail"
      ios="test"
      style={
        Array [
          Object {
            "color": undefined,
            "fontSize": 12,
          },
          Array [
            Object {
              "color": "#000",
              "fontSize": 30,
            },
            Object {
              "color": "#000",
              "fontSize": 30,
            },
            Object {
              "textAlign": "right",
            },
            Object {
              "color": "#FF8214",
            },
          ],
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
        ]
      }
      virtual={undefined}
    >
      ?
    </Text>
  </View>
  <View
    style={
      Object {
        "flex": 0.6,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "paddingLeft": 6,
          },
          Object {
            "fontSize": 16,
          },
          Object {
            "fontFamily": "Kanit",
          },
        ]
      }
    >
      test
    </Text>
  </View>
</View>
`;
